README.md

```diff
@@ -617,130 +621,143 @@ Note that it is also possible to generate a synthetic dataset based on the entir
 
 #### Train the model locally on an M-series Mac or on Linux using the simple pipeline
 
-To train the model locally on your M-Series Mac using our simple pipeline and MLX or on your Linux laptop/deskop using an SFT Trainer:
+- To train the model locally on your M-Series Mac using our simple pipeline and MLX or on your Linux laptop/desktop using an SFT Trainer:
 
-```shell
-ilab model train --pipeline simple
-```
-
-> **Note:** ⏳ This process will take a little while to complete (time can vary based on hardware
-and output of `ilab data generate` but on the order of 5 to 15 minutes)
+   ```shell
+   ilab model train --pipeline simple
+   ```
 
-on a Mac `ilab model train` outputs a brand-new model that is saved in the `<model_name>-mlx-q` directory called `adapters.npz` (in `Numpy` compressed array format). For example:
+   ⏳ This process will take a little while to complete (time can vary based on hardware and output of `ilab data generate` but on the order of 5 to 15 minutes)
 
-```shell
-(venv) $ ls instructlab-merlinite-7b-lab-mlx-q/
-├── adapters-010.npz
-├── adapters-020.npz
-├── adapters-030.npz
-├── adapters-040.npz
-├── adapters-050.npz
-├── adapters-060.npz
-├── adapters-070.npz
-├── adapters-080.npz
-├── adapters-090.npz
-├── adapters-100.npz
-├── adapters.npz
-├── added_tokens.json
-├── config.json
-├── model.safetensors
-├── special_tokens_map.json
-├── tokenizer.json
-├── tokenizer.model
-└── tokenizer_config.json
-```
+   On a Mac `ilab model train` outputs a brand-new model that is saved in the `<model_name>-mlx-q` directory called `adapters.npz` (in `Numpy` compressed array format). For example:
 
-on Linux `ilab model train` outputs a brand-new model that can be served in the `models` directory called `ggml-model-f16.gguf`.
+   ```shell
+   (venv) $ ls instructlab-merlinite-7b-lab-mlx-q/
+   ├── adapters-010.npz
+   ├── adapters-020.npz
+   ├── adapters-030.npz
+   ├── adapters-040.npz
+   ├── adapters-050.npz
+   ├── adapters-060.npz
+   ├── adapters-070.npz
+   ├── adapters-080.npz
+   ├── adapters-090.npz
+   ├── adapters-100.npz
+   ├── adapters.npz
+   ├── added_tokens.json
+   ├── config.json
+   ├── model.safetensors
+   ├── special_tokens_map.json
+   ├── tokenizer.json
+   ├── tokenizer.model
+   └── tokenizer_config.json
+   ```
 
 #### Train the model locally with GPU acceleration
 
-Training has experimental support for GPU acceleration with Nvidia CUDA or AMD ROCm. Please see [the GPU acceleration documentation](./docs/gpu-acceleration.md) for more details. At present, hardware acceleration requires a data center GPU or high-end consumer GPU with at least 18 GB free memory.
+- Training has experimental support for GPU acceleration with Nvidia CUDA or AMD ROCm. Please see [the GPU acceleration documentation](./docs/gpu-acceleration.md) for more details. At present, hardware acceleration requires a data center GPU or high-end consumer GPU with at least 18 GB free memory.
```

Comment:
I wouldn't call it "experimental" at this point :smile: what say you @RobotSail?

Reply:
yeah, not experimental